README.md

```diff
@@ -0,0 +1,28 @@
+# pieces-voice-chat
+
+This is a simple voice chat CLI application that allows users to communicate with [pieces](https://pieces.app) other using their voice.
```

Comment:
Lets make this the following:

A simply cli that allows you to have conversations with [Pieces OS](https://docs.pieces.app/installation-getting-started/pieces-os) using just your voice.